src/app/pages/continue-page/continue-page.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ContinuePageComponent } from './continue-page.component';
+
+describe('ContinuePageComponent', () => {
+  let component: ContinuePageComponent;
+  let fixture: ComponentFixture<ContinuePageComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ ContinuePageComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ContinuePageComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
shouldn't need testing file for final project

Reply:
Angular has a cli command that'll make components for you using "ng generate component" and the testing file comes with it.

Reply:
yep!  we can delete them though :) try to reduce the mass # of files that will be incoming